src/client/src/apps/MainRoute/widgets/spotTile/epics/combineSpotTileEpics.ts

```diff
@@ -22,9 +21,5 @@ export default ({ platform }: ApplicationDependencies) => {
     epics.push(publishPriceUpdateEpic, publishTradeExecutedEpic, closePositionEpic)
   }
 
-  if (platformHasFeature(platform, 'app')) {
-    epics.push(displayCurrencyChartEpic)
-  }
-
```

Comment:
Seeing this I am now wondering if we are doing the right thing in removing this entirely instead of just disabling the feature.

I am not currently sure if the Finsemble platform uses this feature as well. I guess it's easy enough to restore if we decide to pull it back in.